### PR TITLE
[App Service] `az functionapp create`: Add new parameter `--min-replicas` and `--max-replicas` to support minimum and maximum replicas

### PR DESCRIPTION
**Pull request by kaibocai**
**Related command**

`az functionapp create` add two new parameters `--min-replicas` and `--max-replicas`  
`az functionapp config container set` add two new parameters `--min-replicas` and  `--max-replicas`

**Description**

This parameter is required to support create function app running on container app. 

**Testing Guide**

Add unit tests for testing and validation. 

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [X] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [X] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [X] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

**Comment by kaibocai** [comment]
Hi @zhoxing-ms , please review this PR, the CI all pass except the one for PR title, I already fix the title, so it should pass now as well. We want to merge this PR asap. Thank you very much!

File: src/azure-cli/azure/cli/command_modules/appservice/_params.py [hunk 1]
```diff
@@ -391,6 +395,8 @@ def load_arguments(self, _):
                    help='the container registry server username')
         c.argument('registry_password', options_list=['--registry-password', '-p', c.deprecate(target='--docker-registry-server-password', redirect='--registry-password')],
                    help='the container registry server password')
+        c.argument('min_replicas', options_list=['--min-replicas'], type=int, help="The minimum number of replicas when create funtion app on container app", is_preview=True)
+        c.argument('max_replicas', options_list=['--max-replicas'], type=int, help="The maximum number of replicas when create funtion app on container app", is_preview=True)
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/appservice/_params.py:399]: Please do not need to explicitly specify the default option name repeatedly
```suggestion
        c.argument('min_replicas', type=int, help="The minimum number of replicas when create funtion app on container app", is_preview=True)
        c.argument('max_replicas', type=int, help="The maximum number of replicas when create funtion app on container app", is_preview=True)
```

File: src/azure-cli/azure/cli/command_modules/appservice/_params.py [hunk 2]
```diff
@@ -727,6 +733,8 @@ def load_arguments(self, _):
         c.argument('registry_username', options_list=['--registry-username', '-d', c.deprecate(target='--docker-registry-server-user', redirect='--registry-username')], help='The container registry server username.')
         c.argument('registry_password', options_list=['--registry-password', '-w', c.deprecate(target='--docker-registry-server-password', redirect='--registry-password')],
                    help='The container registry server password. Required for private registries.')
+        c.argument('min_replicas', options_list=['--min-replicas'], type=int, help="The minimum number of replicas when create funtion app on container app", is_preview=True)
+        c.argument('max_replicas', options_list=['--max-replicas'], type=int, help="The maximum number of replicas when create funtion app on container app", is_preview=True)
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/appservice/_params.py:737]:
```suggestion
        c.argument('min_replicas', type=int, help="The minimum number of replicas when create funtion app on container app", is_preview=True)
        c.argument('max_replicas', type=int, help="The maximum number of replicas when create funtion app on container app", is_preview=True)
```

**Comment by kaibocai** [comment]
@zhoxing-ms , passed all tests, please review and merge. Thanks.

**Review by kamperiadis — APPROVED** [review]
LGTM!

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**